sensors/uds.py

```diff
@@ -0,0 +1,56 @@
+import RPi.GPIO as GPIO
+import time
+import typing
+import enum
+import random
+
+
+WAKEUP = 0.2
+MISC = 0.00001  # FIXME: What is this delay for?
```

Comment:
To emit a sound wave, the HC-SR04 needs to receive a 5V signal for at least 10µs, or 0.00001s.

Reply:
https://github.com/magley/my-pi-home/pull/3/commits/eadbc344b96336e39034cd384cd573b99628ba5b